src/tiled/automappingutils.cpp

```diff
@@ -42,7 +43,22 @@ void eraseRegionObjectGroup(MapDocument *mapDocument,
         // tile objects. polygons and polylines are not covered correctly by this
```

Comment:
Hmm, it does not look like it to me, because it already says the correct thing is done for tile objects. The patch seems to make the comment more correct than before.